app/views/projects/_form.html.erb

```diff
@@ -293,6 +293,7 @@ databases when reporting vulnerabilities.
           </li>
 
    <%# Note: render() accepts met_suppress: ..., unmet_placeholder:  ... %>
+      <% cache [project, is_disabled, @result] do %>
```

Comment:
I think we should only cache when is_disabled is true.  If it's false, the user is editing it, and then the cache will be useless anyway.

Also: Caching on @result is pointless, it's always nil.